resources/apigateway/service-mesh-settings.yaml

```diff
@@ -0,0 +1,5 @@
+podLabels:
+  sidecar.istio.io/inject: 'true'
+  spire-managed-identity: 'true'
+podAnnotations:
+  inject.istio.io/templates: 'sidecar,spire'
```

Comment:
When Istio is used, this adds an extra sidecar to the Kong pod to enable upstream calls with mTLS.